dashboard/db/schema.rb

```diff
@@ -772,7 +772,7 @@
     t.index ["platform_id"], name: "index_lti_integrations_on_platform_id"
   end
 
-  create_table "lti_user_identities", charset: "utf8", collation: "utf8_unicode_ci", force: :cascade do |t|
+  create_table "lti_user_identities", charset: "utf8mb4", collation: "utf8mb4_unicode_ci", force: :cascade do |t|
```

Comment:
Please, return the encoding back
```suggestion
  create_table "lti_user_identities", charset: "utf8", collation: "utf8_unicode_ci", force: :cascade do |t|
```

Reply:
indeed. why did it do this. hmm. i will ensure and truncate the schema change to the line with the specific column change.

Reply:
The deployment process will force the encoding back to how it should be (staging or test does this), so this change doesn't matter. e.g. https://github.com/code-dot-org/code-dot-org/commit/f273b51f9d1e1d33a061c09c61a135c6c3e43e1f (this is the auto-change that came after my commit)